python/pyspark/ml/tests.py

```diff
@@ -534,6 +534,8 @@ def test_save_load(self):
         cvModel.save(cvModelPath)
         loadedModel = CrossValidatorModel.load(cvModelPath)
         self.assertEqual(loadedModel.bestModel.uid, cvModel.bestModel.uid)
+        for index in range(len(loadedModel.avgMetrics)):
```

Comment:
@holdenk Thanks for suggestion. I used `assert_almost_equal` [here](https://github.com/vectorijk/spark/blob/spark-12810/python/pyspark/ml/tests.py#L561).